src/framework/MOM_domains.F90

```diff
@@ -3,8 +3,9 @@ module MOM_domains
 
 ! This file is part of MOM6. See LICENSE.md for the license.
 
+
 use MOM_array_transform, only : rotate_array
-use MOM_coms, only : PE_here, root_PE, num_PEs, MOM_infra_init, MOM_infra_end
+use MOM_coms, only : PE_here, root_PE, num_PEs, MOM_infra_init, MOM_infra_end, Get_PElist
```

Comment:
It doesn't look like Get_PElist is required anymore. Also, an extra blank line has been inserted.